modules/blocks/verify.js

```diff
@@ -122,7 +124,7 @@ Verify.prototype.verifyBlock = function (block) {
 	// Calculate expected rewards
 	var expectedReward = __private.blockReward.calcReward(block.height);
 
-	if (block.height !== 1 && expectedReward !== block.reward) {
+	if (block.height !== 1 && expectedReward !== block.reward && !_.includes(exceptions.blockRewards, block.id)) {
```

Comment:
Instead of including lodash just for that - we can perform that check like `&& exceptions.blockRewards.indexOf(block.id) === -1`